### PR TITLE
Feature: Display prompt when running Files as admin

### LINKED ISSUE #13209: Feature Request: Modal for disabled drag'n'drop in Admin Mode
### What feature or improvement do you think would benefit Files?

#13134 disabled several instance of drag and drop through the app to prevent a crash when running admin mode. (See #12390)

Regardless, there is no way for the user to see it's an intended behavior. A modal should be displayed when the user tries to drag and drop in those situations, with a `don't show again` checkbox.

The disable situations are:
- Drag and drop in the main view
- Drag and drop in the tag edition settings tab
- Drag and drop to reorder favorites

### Requirements

- This proposal will enhance the user experience by providing feedback.

### Files Version

2.5.22 

### Windows Version

10.0.19045.3324

### Comments

_No response_

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #13209...

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app as admin
   2. Confirm that clicking "don't show again" works as expected

**Screenshots (optional)**
image

### CODE REVIEW

**Comment by yaira2** [comment]
@chingucoding are the automation tests ran as admin?

File: src/Files.App/Views/MainPage.xaml.cs [hunk 1]
```diff
@@ -273,6 +285,18 @@ private void Page_Loaded(object sender, RoutedEventArgs e)
 			FindName(nameof(TabControl));
 			FindName(nameof(NavToolbar));
 
+			// Notify user that drag and drop is disabled
+			// ToDo put this in a StartupPromptService
+			if
+			(
+				isAppRunningAsAdmin == true &&
```
[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Views/MainPage.xaml.cs]: Thanks for the suggestion, I applied this change in [5386963](https://github.com/files-community/Files/pull/13221/commits/538696337ba7c76bddac910105fa0e9a9d34ff1d)

**Comment by chingucoding** [comment]
> @chingucoding are the automation tests ran as admin?

 @yaira2 Apparently we are running as Admin: "Windows virtual machines are configured to run as administrators with User Account Control (UAC) disabled" [Documentation for GitHub Actions (they use the same images as Azure DevOps)](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners)

**Pull request merged**